### ISSUE TITLE
Missing dependency on `m4`

### ISSUE THREAD

**Issue by Shoeboxam** [r-universe-org/help#287]
Is there any advice for how I should approach resolving a missing system dependency on R-Universe? I have a dependency that requires GNU m4 to be installed on the Windows image. Am I expected to try to do my own vendored install within Makevars?

Failing workflow:
https://github.com/r-universe/opendp/actions/runs/5918601735/job/16055386395

In my usual CI, I'm able to prepare Windows binaries when just adding this one dependency:
https://github.com/opendp/opendp/blob/main/.github/workflows/build.yml#L54-L58

Thank you!

**Comment by jeroen** [comment]
We use the build environment from rtools43, just like cran. Do you need m4 to build the r package?

**Comment by Shoeboxam** [comment]
Right, I need m4 to build the R package, unfortunately. I assume you'd rather not deviate from CRAN's environment. Is it possible to try to install it as part of the configure or Makevars script? 

Otherwise, it seems like I would need to avoid R-universe until I can cut the dependency out.

**Comment by jeroen** [comment]
On r-universe, the R source package gets built on Linux. Can you do the autoconf step when building the source package, such that you dont need m4 on Windows?

**Comment by jeroen** [comment]
Ill help you with your use case when im back from vacation next week.

**Comment by Shoeboxam** [comment]
I'm not sure if I can do the autoconf step when building sources, because the m4 dependency itself comes from another dependency. I think I'd have to find a way to patch the build process for that dependency. I unfortunately will not be able to work on this right now, as I'm traveling for ~4 weeks.
